server/Arcadia.Assistant.Server.Console/appsettings.json

```diff
@@ -42,7 +42,7 @@
       "SickLeaveApproved": {
         "NotificationRecipient": "",
         "NotificationSender": "",
-        "Subject": "Approved sick leave",
+        "Subject": "Approved sick leave for {employee}",
```

Comment:
Please, replace Testing team with 'Arcadia Assistant' everywhere in templates :)
And just 'Sick leave: {employee}'

Reply:
Done.